kratos/utilities/cpp_tests_utilities.cpp

```diff
@@ -69,8 +71,48 @@ void Create2DGeometry(
 /***********************************************************************************/
 /***********************************************************************************/
 
+void Create2DGeometryDummyElements(ModelPart& rModelPart)
```

Comment:
```suggestion
void CreateTestModelPartTriangle2D3N(ModelPart& rModelPart)
```
I was thinking in something of this sort...

Reply:
But in that case is ambiguous with the other Create functions (also generate triangles)